src/FsUnit.Xunit/CustomMatchers.fs

```diff
@@ -57,8 +57,8 @@ let throw(t: Type) =
             try
                 testFunc()
                 false
-            with
-            | ex -> t.IsAssignableFrom(ex.GetType())
+            with ex ->
+                t.IsAssignableFrom(ex.GetType())
```

Comment:
@CaptnCodr  do you like such formatting? never saw it  formated  this way

Reply:
I'm wondered about that is actual legal. But I kinda like it. 
That is also mentioned here: https://docs.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-trywith-expressions

Reply:
Then it is good to go